applications/GeoMechanicsApplication/custom_geometries/README.md

```diff
@@ -0,0 +1,18 @@
+# Custom Geometries
+Geometries are responsible for calculating several geometric properties, such as the normal vector, shape function values at a given position and the (inverse/determinant of the) Jacobian. This folder contains custom geometries that are used in the GeoMechanicsApplication.
```

Comment:
As far as I remember, the inverse of jacobian is not defined for this geometry!

Reply:
I think it is not necessary to mention that the inverse Jacobian is not calculated/defined for this geometry. It is too much details. So, maybe it is good to leave the text as it is.